compute/metadata/metadata.go

```diff
@@ -114,24 +114,35 @@ var (
 )
 
 // OnGCE reports whether this process is running on Google Compute Platforms.
-// NOTE: True returned from `OnGCE` does not guarantee that the metadata server
-// is accessible from this process and have all the metadata defined.
+//
+// Deprecated: Please use the context aware variant [OnGCEWithContext].
```

Comment:
We're not yet ready to deprecate `OnGCE`. Can you please revert these lines, restoring the original note?